operators/jaeger/ci.yaml

```diff
@@ -1,3 +1,6 @@
 ---
 # Use `replaces-mode` or `semver-mode`. Once you switch to `semver-mode`, there is no easy way back.
-updateGraph: replaces-mode
\ No newline at end of file
+updateGraph: replaces-mode
+reviewers:
+  - jpkrohling
```

Comment:
@jpkrohling  this might be the place to set the reviewer. 

@rkukura please add here @rubenvp8510 as he is now the main maintainer of the Jaeger operator.

Reply:
This merged prior to seeing the comment. I added @pavolloffay, @rubenvp8510 and @rkukura in community-operators before that one merged, and we can update community-operators-prod on the next release.